src/WinRPM.jl

```diff
@@ -451,21 +452,23 @@ const exe7z = iswindows() ? joinpath(BINDIR, "7z.exe") : "7z"
 function do_install(package::Package)
     name = names(package)
     source, path = rpm_url(package)
-    info("Downloading: ", name)
+    @info("Downloading: ", "$source/$path")
     data = download("$source/$path")
     if data[2] != 200
-        info("try running WinRPM.update() and retrying the install")
+        @info("try running WinRPM.update() and retrying the install")
         error("failed to download $name $(data[2]) from $source/$path.")
     end
     cache = getcachedir(source)
-    path2 = joinpath(cache,escape(path))
+    path2 = joinpath(cache, escape(path))
     open(path2, "w") do f
         write(f, data[1])
     end
-    info("Extracting: ", name)
+    @info("Extracting: ", name)
     cpio = splitext(path2)[1]*".cpio"
+    cpio = replace(cpio, "noarch%2F", "")
```

Comment:
I'm not really sure why this was suddenly needed... Are these related to changes in the binary repositories?

Reply:
No. See #141. Did you test this on 0.6, actually installing anything from this branch?

Reply:
Oh I guess I should just close this in favor of #141?

Reply:
Well, better rebase this one against #141 I guess